generators/generator-microsoft-bot-command-list/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@microsoft/generator-microsoft-bot-command-list",
+  "version": "1.0.0",
+  "description": "Preview command list bot for TESTING ONLY",
+  "main": "./generators/app/templates/index.js",
+  "dependencies": {
+    "generator-adaptive-bot": "^1.0.0-preview.9",
```

Comment:
Make sure to change to @microsoft/generator-microsoft-bot-adaptive and utilize latest published package version.